Source/Artifacts/Artifacts.csproj

```
@@ -4,6 +4,11 @@
   <PropertyGroup>
     <AssemblyName>Dolittle.SDK.Artifacts</AssemblyName>
   </PropertyGroup>
+    
+  <ItemGroup>
+    <PackageReference Include="System.Reactive" Version="4.4.1" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="$(MicrosoftExtensionsVersion)" />
+  </ItemGroup>
```

```suggestion
  <ItemGroup>
    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="$(MicrosoftExtensionsVersion)" />
  </ItemGroup>
```
Reactive not used anymore and we can just use the abstractions logging.